src/localizers/moment.js

```diff
@@ -36,6 +39,7 @@ export let formats = {
   agendaHeaderFormat: dateRangeFormat,
 
   agendaDateFormat: 'ddd MMM DD',
+  agendaSingleTimeFormat: singleTimeFormat,
```

Comment:
Can we use the format below `agendaTimeFormat` instead of adding a new one? It should be what we want here 👍

Reply:
Good catch, done!